pkg/logger/logger.go

```diff
@@ -13,115 +15,103 @@ const (
 	RESET  = "\x1b[33;0m"
 )
 
-type LogMessage struct {
-	Level   string
-	Message string
-}
-
 // Logger struct
 type Logger struct {
 	Verbosee bool
 	Debugg   bool
 	NoColor  bool
+	writer   io.Writer
+}
+
+func GetLogger() Logger {
+	logger := Logger{}
+	logger.Init()
+	return logger
+}
+
+// initialize the Logger to write to standard output
+func (l *Logger) Init() {
+	l.writer = os.Stdout
+}
+
+// ensure the Logger is initialized on first print
+func (l *Logger) LazyInit() {
```

Comment:
```suggestion
func (l *Logger) lazyInit() {
```

I don't think you need it to be exported

Reply:
You're totally right. fixed.